lib/actual_db_schema/commands/base.rb

```diff
@@ -4,6 +4,10 @@ module ActualDbSchema
   module Commands
     # Base class for all commands
     class Base
+      def initialize(context: nil)
```

Comment:
```suggestion
      def initialize(context)
```

Why it's an optional param? Isn't it mandatory?